vpd-manager/src/parser.cpp

```diff
@@ -198,6 +204,10 @@ int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
         l_bytesUpdatedOnHardware = constants::FAILURE;
     }
 
+    // Disable Reboot Guard. We do not need to check the return value here as
+    // there is no action to take incase DisableRebootGuard fails here.
+    dbusUtility::DisableRebootGuard();
```

Comment:
Caller should be aware that this call failed and should log the PEL accordingly.